### DISCUSSION TITLE
Is there a keyboard shortcut to re-invoke the link creation?

### DISCUSSION THREAD

**Discussion by pmagwene** [svsool/vscode-memo#523] [Q&A] [answered]
If I start to create a link, but have a misspelling and backspace, the link creation dialog goes away. Is there a keyboard shortcut to re-invoke the dialog? Or is there an exported command I could setup my on shortcut for? 

I Tried grokking the source, but am new to TypeScript and VS Code extension structure so it wasn't clear to me where I should be looking.

**Comment by svsool** [comment, marked as answer]
Hi,

Autocomplete triggered either by typing `[` character (see [here](https://github.com/svsool/vscode-memo/blob/0a777547edd0870eab5446d4d161f01ed9c544e9/src/features/completionProvider.ts#L145)) or by using `^Space` (ctrl + space) shortcut described in the [documentation](https://code.visualstudio.com/docs/editor/intellisense#_intellisense-features).

In command palette this command is called "Trigger Suggest" and you can rebind it to any keyboard shortcut if needed using VS Code `keybindings.json` and `editor.action.triggerSuggest` command key.

![image](https://user-images.githubusercontent.com/1452141/151654266-3841d885-9275-492f-a570-f41a2f9d5d64.png)
